doc/read-the-docs-site/tutorials/english-auction-example.rst

```diff
@@ -0,0 +1,344 @@
+The effect of a transaction is to consume inputs and produce new outputs.
+
+In this English auction example, the item owner, Alice, wants to auction an NFT (Non-fungible token), a native token on Cardano that exists only once.
+An NFT can represent a piece of digital art or possibly a real-world asset.
+
+The general logic of the auction takes into account the following parameters:
+
+* the owner of the token,
+* the token itself,
+* a minimal bid,
+* a higher bid, and
+* a deadline.
+
+At the outset, Alice creates a transaction for the English auction that has a EUTXO at the script output.
+
+The value of the EUTXO is the NFT, and the datum initially is nothing.
+Later it will be the highest bidder and the highest bid. But at this stage, a bid has not yet taken place.
+
+   *On the Cardano blockchain you can’t have a EUTXO that contains only native tokens. A EUTXO always has to be accompanied by some Ada, but for the sake of simplicity we will temporarily ignore that requirement for now.*
+
+Bidder transactions
+----------------------
+
+In this example, Bob, the first bidder, wants to bid 100 Ada for Alice's NFT.
+In order to do this, Bob creates a transaction that has two inputs and one output.
+
+The two inputs are:
+
+* the auction EUTXO
+* Bob's bid of 100 Ada
+
+Datum
+~~~~~~~~~
+
+The output is at the output script, but now the value and the datum have changed.
+Previously the datum was nothing but now it has the value of: (Bob, 100).
+
+Now there is not only the NFT in the EUTXO, but also the 100 Ada bid.
+
++----------------------+-------------------+
+| 1st bid inputs       | 1st bid outputs   |
++======================+===================+
+| The auction EUTXO    | Bob               |
++----------------------+-------------------+
+| Bob's bid of 100 Ada | 100 Ada           |
++----------------------+-------------------+
+
+Redeemer
+~~~~~~~~~~~~
+
+As a redeemer, in order to unlock the original auction EUTXO, we use something called `Bid`.
+This is an algebraic data type. There will be other values as well.
+
+The auction script will check that all the conditions are satisfied.
+In this example, the script has to check these three conditions:
+
+1. The bid happens before the deadline.
+2. The bid is high enough.
+3. The correct inputs and outputs are present (meaning that the auction is an output containing the NFT and it has the correct datum).
+
+A second bidder
+~~~~~~~~~~~~~~~~~~~~
+
+Next, a second bidder, Charlie, wants to outbid Bob. Charlie wants to bid 200 Ada.
+
+Charlie will create another transaction, this one having two inputs and two outputs.
+The two inputs are:
+
+* the auction EUTXO
+* Charlie's bid of 200 Ada
+
+The two outputs are:
+
+* the updated auction EUTXO
+* a EUTXO that returns Bob's bid of 100 Ada
+
+.. note::
+    To be more clear, technically, the auction EUTXO is not getting updated because nothing ever changes.
```

Comment:
```suggestion
    To be more clear, technically, the auction UTXO is not getting updated because UTXOs are immutable - they always refer to one specific output of one specific transaction.
```